Service/DeveloperOverheidService.php

```diff
@@ -297,19 +331,88 @@ public function getComponent(string $id): ?array
      *
      * @return ?ObjectEntity
      */
-    public function handleRepositoryArray(array $repository, ?Entity $repositoryEntity = null, ?Source $developerOverheidSource = null): ?ObjectEntity
+    public function handleRepositoryArray(array $repository, ?Source $developerOverheidSource = null): ?ObjectEntity
```

Comment:
This is oke, but we could probably choose to not pass the $developerOverheidSource to this function.
But use a get...Source() function for this